File: src/main/java/com/mbayou/kick4k/chat/PostChatMessageResponse.java
package com.mbayou.kick4k.chat;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;

public class PostChatMessageResponse {
    private final Boolean isSent;
    private final String messageId;

    @JsonCreator
    public PostChatMessageResponse(@JsonProperty("is_sent") Boolean isSent,
                                   @JsonProperty("message_id") String messageId) {
        this.isSent = isSent;
        this.messageId = messageId;
    }

    public Boolean getSent() {
        return isSent;
    }

    public String getMessageId() {
        return messageId;
    }
}

File: src/main/kotlin/com/mbayou/kick4k/chat/PostChatMessageResponse.kt
package com.mbayou.kick4k.chat

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty

data class PostChatMessageResponse @JsonCreator constructor(
    @JsonProperty("is_sent") val isSent: Boolean?,
    @JsonProperty("message_id") val messageId: String?,
)
